editor/editor_properties.cpp

```diff
@@ -2703,7 +2703,7 @@ void EditorPropertyNodePath::_update_menu() {
 void EditorPropertyNodePath::_menu_option(int p_idx) {
 	switch (p_idx) {
 		case ACTION_CLEAR: {
-			emit_changed(get_edited_property(), NodePath());
+			emit_changed(get_edited_property(), Variant());
```

Comment:
`Variant()` breaks clearing for `@export var array: Array[NodePath]`.
```suggestion
			if (editing_node) {
				emit_changed(get_edited_property(), Variant());
			} else {
				emit_changed(get_edited_property(), NodePath());
			}
```

Reply:
Thanks for catching this. I thought there was an implicit conversion from nil to the null NodePath.